x-pack/plugins/security_solution/cypress/integration/detection_alerts/alerts_details.spec.ts

```diff
@@ -80,4 +83,21 @@ describe('Alert details with unmapped fields', () => {
         expect($tableContainer[0].scrollLeft).to.equal(0);
       });
   });
+
+  it('Opens a new timeline investigation', () => {
+    openOverview();
+
+    cy.get(SUMMARY_VIEW_PREVALENCE_CELL)
+      .invoke('text')
+      .then((alertCount) => {
+        // Click on the first button that lets us investigate in timeline
+        cy.get(ALERT_FLYOUT).find(SUMMARY_VIEW_INVESTIGATE_IN_TIMELINE_BUTTON).click();
+
+        // Make sure a new timeline is created and opened
+        cy.get(TIMELINE_TITLE).should('contain.text', 'Untitled timeline');
+
+        // The alert count in this timeline should match the count shown on the alert flyout
+        cy.get(QUERY_TAB_BUTTON).should('contain.text', alertCount);
```

Comment:
Fields that typically have just one value, like `process.name`, consistently display the same alert counts in both the `Alert prevalence` column in the flyout, and in Timeline.

Fields that have more than one value sometimes display different counts between the flyout and Timeline.

Example: The `process.args` in the sample JSON below contains two values:

```json
    "process": {
      "args": [
        "\"C:\\lsass.exe\"",
        "--jyf"
      ],
      "name": "lsass.exe",
    },
```

In the video below:

- The `process.args` field displays `570` in the `Alert prevalence` column in the flyout
- Timeline displays a count of `285` alerts when the `Investigate in timeline` action is invoked

https://user-images.githubusercontent.com/4459398/167693128-52a0024e-4f71-4308-a8f5-c67050aed8f7.mov

Reply:
Per a chat with @michaelolo24, we'll track this observation in a separate issue: https://github.com/elastic/kibana/issues/131967